docs/notes-on-frameworks.md

```diff
@@ -1,22 +0,0 @@
-# Notes on Frameworks
```

Comment:
Some of this content should serve as a baseline for a similar document on using frameworks that should live in the Development repo.